src/sfctl/helps/app.py

```diff
@@ -94,6 +94,10 @@
           type: bool
           short-summary: Forcefully restart processes during upgrade even
             when the code version has not changed
+        - name: --sort-order
+          type: string
+          short-summary: "Defines the order in which an upgrade proceeds through the cluster.
```

Comment:
Please remove the " around the short summary

Reply:
The swagger specs are much clearer IMO. Can we add some more details here? It's okay to be a bit verbose. Referring to this:

    - value: Default
      description: Indicates that the default sort order (as specified in cluster manifest) will be used. The value is 1.
    - value: Numeric
      description: Indicates that forward numeric sort order (UD names sorted as numbers) will be used. The value is 2.
    - value: Lexicographical
      description: Indicates that forward lexicographical sort order (UD names sorted as strings) will be used. The value is 3.
    - value: ReverseNumeric
      description: Indicates that reverse numeric sort order (UD names sorted as numbers) will be used. The value is 4.
    - value: ReverseLexicographical
      description: Indicates that reverse lexicographical sort order (UD names sorted as strings) will be used. The value is 5.

Thanks!